packages/db-base/src/lib/inMemFileDB.ts

```diff
@@ -34,31 +34,40 @@ import { createGzip } from 'zlib';
 // };
 //
 
-interface ConnectionOptions {
-    // relative path to the data dir
+export interface ConnectionOptions {
+    pass?: string;
+    sentinelName?: string;
+    /** array on sentinel */
+    host: string | string[];
+    /** array on sentinel */
+    port: number | number[];
+    options: Record<string, any>;
+    maxQueue?: number;
+    enhancedLogging?: boolean;
     backup?: BackupOptions;
+    /** relative path to the data dir */
     dataDir: string;
 }
 
 type ChangeFunction = (id: string, state: any) => void;
 
-interface DbStatus {
+export interface DbStatus {
     type: string;
     server: boolean;
 }
 
 interface BackupOptions {
-    // deactivates backup if false
+    /** deactivates backup if true */
     disabled: boolean;
```

Comment:
Maybe 
```suggestion
    disabled?: true;
```
would be a better type, but not sure.

Reply:
Everything fixed except this one, not sure whats the benefit here.

Reply:
The intention would be clearer if the desired options are
- property not given and
- `disabled: true`
since explicitly setting `disabled: false` doesn't make lots of sense when that is the default.

But that doesn't really matter.